daemon/src/GHCSpecter/UI/ConcurReplica/Types.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module GHCSpecter.UI.ConcurReplica.Types
+  ( IHTML (..),
+    project,
+    embed,
+  )
+where
+
+import Concur.Core (SuspendF (..), Widget (Widget, step))
+import Control.Monad.Free (hoistFree)
+import Control.ShiftMap (ShiftMap (..))
+import Replica.VDOM (HTML)
+
+-- | Left: no need for update, Right: need for update
+newtype IHTML = IHTML {unIHTML :: Either HTML HTML}
```

Comment:
IHTML has additional tag about whether one wants to bypass DOM update. 
As mouse move events are too frequent, and most of the handling action is just  to update internal state, not DOM changes, it is desired to be marked non-update and bypass expensive websocket diff update steps.

Reply:
added comment